packages/styling/src/interfaces/IFontStyles.ts

```diff
@@ -4,16 +4,16 @@ import { IRawStyle } from '../interfaces/index';
  * UI Fabric font set.
  */
 export interface IFontStyles {
```

Comment:
The final ITheme probably should have required values, like you have. But we need a way such that we can have a partial of partials? How do we do that? I think using `type` you could define IPartialTheme type, with keyof ITheme optionaled to a partial of the type, maybe.